Utilities/GetPSGalleryItemsForAuthor.ps1

```diff
@@ -0,0 +1,100 @@
+﻿
+<#PSScriptInfo
+
+.VERSION 0.1.0
+
+.GUID 788b94eb-f6f0-400d-b259-2e15c3f102ee
+
+.AUTHOR J. Keith Bankston
+
+.COMPANYNAME Microsoft
+
+.COPYRIGHT Microsoft, 2018
+
+.TAGS GalleryUtility,GDPR
+
+.LICENSEURI 
```

Comment:
Please specify LicenseUri and ProjectUri values.

Reply:
Definitely need ProjectURI, added in pending PR. LicenseURI I was not planning to add, as the PSGallery built-in license is sufficient.